server/serializer/subscription.go

```diff
@@ -25,7 +25,9 @@ type SubscriptionPayload struct {
 type SubscriptionResponse struct {
 	SysID              string `json:"sys_id"`
 	UserID             string `json:"user_id"`
+	UserName           string `json:"userName"`
```

Comment:
```suggestion
	UserName           string `json:"-"`
```
Same for the channelName.